### PR TITLE
added mutli chain nfts structure

### PR DESCRIPTION
**Pull request by poocart**
Added support for multi chain NFTs. No service hooked yet for other chains than Ethereum mainnet.

### CODE REVIEW

File: src/actions/assetsActions.js [hunk 1]
```diff
@@ -125,11 +124,11 @@ export const sendAssetAction = (
     const accountId = getAccountId(activeAccount);
     const accountAddress = getAccountAddress(activeAccount);
 
-    const accountCollectibles = collectibles[accountId] || [];
+    const accountCollectiblesOnChain = accountCollectiblesSelector(getState())?.[chain] ?? [];
```
Comment (IAmKio) [inline src/actions/assetsActions.js:127]:
```suggestion
    const accountCollectiblesOnChain = accountCollectiblesSelector(getState())?.[chain] ?? [];
```

Just curious here - are we evaluating the output of getState() as a whole to be truthy or falsey here?

File: src/reducers/collectiblesReducer.js [hunk 2]
```diff
@@ -82,12 +86,24 @@ const collectiblesReducer = (
...
+
   switch (action.type) {
-    case UPDATE_COLLECTIBLES:
+    case SET_COLLECTIBLES:
       return {
         ...state,
         data: action.payload || {},
       };
+    case SET_ACCOUNT_COLLECTIBLES:
+      ({ accountId } = action.payload);
```
Comment (IAmKio) [inline src/reducers/collectiblesReducer.js:98]: Do we need the parenthesis here?

File: src/reducers/collectiblesReducer.js [hunk 3]
```diff
@@ -100,8 +116,8 @@ const collectiblesReducer = (
         updatingTransaction: action.payload,
       };
     case ADD_COLLECTIBLE_HISTORY_TRANSACTION:
-      const { accountId } = action.payload;
-      const accountCollectibles = state.data[accountId] ?? [];
+      ({ accountId } = action.payload);
```
Comment (IAmKio) [inline src/reducers/collectiblesReducer.js:119]: Ahh i can see what you're achieving here

File: src/services/dataMigration/collectibles.js [hunk 4]
```diff
@@ -1,9 +1,48 @@
...
+
+// actions
+import { saveDbAction } from 'actions/dbActions';
+
+// types
+import type { Dispatch } from 'reducers/rootReducer';
+
+
+export default function (storageData: Object, dispatch: Dispatch) {
+  const collectibles = storageData?.collectibles?.collectibles ?? {}; // not a mistype
```
Comment (IAmKio) [inline src/services/dataMigration/collectibles.js:29]: 😂 NFT inception

**Review by IAmKio — APPROVED** [review]
Just a few questions but all good!

File: src/reducers/collectiblesReducer.js (hunk 2 above)
Comment (poocart) [reply src/reducers/collectiblesReducer.js:98]: Yeah, either we need to kill that eslint rule or keep applying like this 👀

File: src/actions/assetsActions.js (hunk 1 above)
Comment (poocart) [reply src/actions/assetsActions.js:127]: In case the output of `accountCollectiblesSelector` is falsey then it falls back to array

Comment (IAmKio) [reply src/actions/assetsActions.js:127]: Got it - cheers!

**Pull request merged**